tools/wasp-cli/cli/config/config.go

```diff
@@ -78,7 +78,11 @@ func L1FaucetAddress() string {
 }
 
 func GetToken(node string) string {
-	return viper.GetString(fmt.Sprintf("authentication.wasp.%s.token", node))
+	token := viper.GetString(fmt.Sprintf("authentication.wasp.%s.token", node))
+	if token == "" {
+		log.Fatalf("wasp node '%s' doesn't exist in config", node)
```

Comment:
This error message can be misleading. The wasp node may exist but not have a `token`.

```suggestion
		log.Fatalf("token for wasp node '%s' not defined in config", node)
```

Reply:
done